community/detectors/apache_activemq_cve_2023_46604/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'java-library'
+}
+
+description = 'Tsunami Apache ActiveMQ RCE (CVE-2023-46604) VulnDetector plugin.'
+group 'com.google.tsunami'
+version '0.0.1-SNAPSHOT'
+
+
+repositories {
+    maven { // The google mirror is less flaky than mavenCentral()
+        url 'https://maven-central.storage-download.googleapis.com/repos/central/data/'
+    }
+    mavenCentral()
+    mavenLocal()
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+
+    jar.manifest {
+        attributes('Implementation-Title': name,
+                'Implementation-Version': version,
+                'Built-By': System.getProperty('user.name'),
+                'Built-JDK': System.getProperty('java.version'),
+                'Source-Compatibility': sourceCompatibility,
+                'Target-Compatibility': targetCompatibility)
+    }
+
+    javadoc.options {
+        encoding = 'UTF-8'
+        use = true
+        links 'https://docs.oracle.com/javase/8/docs/api/'
+    }
+
+    // Log stacktrace to console when test fails.
+    test {
+        testLogging {
+            exceptionFormat = 'full'
+            showExceptions true
+            showCauses true
+            showStackTraces true
+        }
+        maxHeapSize = '1500m'
+    }
+}
+
+ext {
+    tsunamiVersion = 'latest.release'
+    junitVersion = '4.13'
+    mockitoVersion = '2.28.2'
+    truthVersion = '1.0.1'
+    okhttpVersion = '3.12.0'
+    apacheActiveMqClientVersion = '5.18.2'
+    springframeworContextSupportVersion = '3.2.2.RELEASE'
+}
+
+dependencies {
+    implementation "com.google.tsunami:tsunami-common:${tsunamiVersion}"
+    implementation "com.google.tsunami:tsunami-plugin:${tsunamiVersion}"
+    implementation "com.google.tsunami:tsunami-proto:${tsunamiVersion}"
+    implementation 'com.google.googlejavaformat:google-java-format:1.13.0'
+    implementation("org.springframework:spring-context-support:${springframeworContextSupportVersion}")
+    implementation "org.apache.activemq:activemq-client:${apacheActiveMqClientVersion}"
```

Comment:
This dependency is not included in the final jar, therefore the plugin fails at runtime. Add a `jar` directive to include this dependency in the final archive.

Reply:
Okey